scripts/deploy.sh

```
@@ -0,0 +1,179 @@
+    local active_port=$(get_active_port)
+    if [ "$active_port" == "$BLUE_PORT" ]; then
+        echo "$GREEN_PORT"
+    else
+        echo "$BLUE_PORT"
+    fi
+}
+
+# 특정 포트의 프로세스 종료
+kill_app_on_port() {
+    local port=$1
+    local pid=$(lsof -ti:$port)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)